templates/typesetting/elements/dashboard.html

```diff
@@ -2,20 +2,27 @@
 {% user_has_role request 'typesetter' as typesetter %}
 {% user_has_role request 'proofreader' as proofreader %}
 {% user_has_role request 'editor' as editor %}
+{% user_has_role request 'production' as production %}
 
 <div class="large-4 columns end" data-equalizer-watch>
     <div class="box">
         <div class="title-area">
             <h2>Typesetting Plugin</h2>
         </div>
         <div class="content">
+            {% if production or editor %}
             <p>
                 There are currently {{ element.articles.count }} articles in the Typesetting stage.
             </p>
+            {% elif not typesetter %}
+                <p>
+                    Any proofreading tasks you are assigned can be accessed using the button below.
+                </p>
+            {% endif %}
```

Comment:
Shouldn't this match the check below? (i.e.: `{% if proofreader or is_author %}`
Just to avoid problems in the very unlikely scenario were someone has an editor/proofreader/author and a typesetter role.